docs/spec_draft.md

```diff
@@ -483,6 +484,59 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.cholesky
+
+### Semantics
+
+Computes the Cholesky decomposition of a batch of symmetric (Hermitian) positive
+definite matrices.
+
+If `lower` is `true`, it computes the lower-triangular matrices `l` such that
+$a = l \cdot l^{T}$. If `lower` is `false`, it computes the upper-triangular
```

Comment:
In the HLO spec, it it says 
$a = l \cdot l^{T}$
with `l` being a matrix. I believe the assumption is that `a` is also a matrix. 

We can define the similar formula with `l` being a tensor with R (= rank) >=2 using the StableHLO versions of the `transpose` and `dot_general` ops.

$l^{T}$ could defined as `stabhlehlo.transpose` on `l` with `permutations = [0, 1, ..., R-1, R-2]`.

$l \cdot l^{T}$, is defined as `stablehlo.dot_general` with
 - `lhs_batching_dims = rhs_batching_dims = {i : 0 <= i < R-2}`
 -  `lhs_contracting_dims = [R-1]`
 -  `rhs_contracting_dims = [R-2]`

Reply:
Yes, that is a correct intuition to derive `a` from `result`.